### PR TITLE
[exporter/pulsarexporter] Update stability to alpha.

### PR DESCRIPTION
**Pull request by tjiuming**
Update pulsarexporter Stability to alpha.

### CODE REVIEW

**Comment by tjiuming** [comment]
@dmitryax  PTAL

File: exporter/pulsarexporter/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 | Status                   |                       |
 | ------------------------ |-----------------------|
-| Stability                | [in-development]      |
+| Stability                | [alpha]               |
 | Supported pipeline types | traces, logs, metrics |
-| Distributions            | none                  |
+| Distributions            | [contrib]             |
```
Comment (dmitryax) [inline exporter/pulsarexporter/README.md:7]: The exporter is not in the contrib distro. Please add it to https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml first

**Comment by dmitryax** [comment]
> do I need a change log file?

No, I added the "Skip Changelog" label

File: exporter/pulsarexporter/README.md (hunk 1 above)
Comment (tjiuming) [reply exporter/pulsarexporter/README.md:7]: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/184 PTAL

**Review by dmitryax — APPROVED** [review]

**Pull request merged**